pycromanager/test/test_MDA_events.py

```diff
@@ -84,6 +84,32 @@ def test_time_points():
     ]
     assert expected == multi_d_acquisition_events(num_time_points=5, time_interval_s=10)
 
+    expected = [
+        {"axes": {"time": 0}, "min_start_time": 0},
+        {"axes": {"time": 1}, "min_start_time": 10},
+        {"axes": {"time": 2}, "min_start_time": 20},
+        {"axes": {"time": 3}, "min_start_time": 30},
+        {"axes": {"time": 4}, "min_start_time": 40},
+    ]
+    assert expected == multi_d_acquisition_events(num_time_points=5, time_interval_s=[0, 10, 10, 10, 10])
+
+    expected = [
+        {"axes": {"time": 0}, "min_start_time": 0},
+        {"axes": {"time": 1}, "min_start_time": 10},
+        {"axes": {"time": 2}, "min_start_time": 30},
+        {"axes": {"time": 3}, "min_start_time": 30},
+        {"axes": {"time": 4}, "min_start_time": 40},
+    ]
+    assert expected == multi_d_acquisition_events(num_time_points=5, time_interval_s=[0, 10, 20, 0, 10])
+
+    expected = [
+        {"axes": {"time": 0}, "min_start_time": 0},
+        {"axes": {"time": 1}, "min_start_time": 10},
+        {"axes": {"time": 2}, "min_start_time": 20},
+        {"axes": {"time": 3}, "min_start_time": 30},
+        {"axes": {"time": 4}, "min_start_time": 20},
+    ]
+    assert expected == multi_d_acquisition_events(num_time_points=5, time_interval_s=[0, 10, 10, 10, -10])
```

Comment:
Should we require that `time_interval_s` is non-negative? pycromanager executes events in order, so here the acquisition engine will execute event `{"axes": {"time": 2}}`, wait 10 seconds, execute event `{"axes": {"time": 3}}` and then execute event `{"axes": {"time": 4}}` right away. I don't think there is ever a case where a negative `time_interval_s` will have any effect.

Reply:
Yes I also think that it won't cause a problem if its negative. Maybe just a warning to let the user know